ui/src/logs/components/LogViewerChart.tsx

```diff
@@ -0,0 +1,33 @@
+import React, {PureComponent} from 'react'
+import LineGraph from 'src/shared/components/LineGraph'
+import {DEFAULT_LINE_COLORS} from 'src/shared/constants/graphColorPalettes'
+
+import {TimeRange} from 'src/types'
+
+interface Props {
+  onZoom: (lower: string, upper: string) => void
+  timeRange: TimeRange
+  data: object[]
```

Comment:
is the shape of this TBD, hence the type annotation? or, if it's a known type, could you specify it further?